frontend/packages/api_client/src/apis/V2Api.ts

```diff
@@ -584,7 +584,9 @@ export class V2Api extends runtime.BaseAPI {
      * Get all instances owned by user
      */
     async instancesOpenedxConfigListRaw(): Promise<runtime.ApiResponse<Array<OpenEdXInstanceConfig>>> {
-        const queryParameters: runtime.HTTPQuery = {};
+        const queryParameters: runtime.HTTPQuery = {
+            fail_if_staff: true,
```

Comment:
You shouldn't change files in packages/api_client directly, they are automatically build from python sources with the `scripts/update-api-client.sh` script. We should remove this from this file. Since the v2 api is currently only used from the frontend, I think we can just remove the `fail_if_staff` parameter and always fail when accessed by a staff user.

By the way while examining how the client is built, I noticed that [these two lines in the build script](https://github.com/open-craft/opencraft/blob/3707c93680532a19068fbba4014d149df8159b98/frontend/scripts/update-api-client.sh#L4-L5) are trying to remove wrong files. It should be `packages/api_client/src/apis/*.ts` and `packages/api_client/src/models/*.ts`. Could you add that change to this PR?

Reply:
Ah, didn't notice the warning at the top - have removed the query param, and updated the buildscript.